Makefile

```diff
@@ -661,7 +571,7 @@ bazel-crossbuild-protokube:
 
 .PHONY: bazel-crossbuild-protokube-image
 bazel-crossbuild-protokube-image:
-	bazel build ${BAZEL_CONFIG} --platforms=@io_bazel_rules_go//go/toolchain:linux_amd64 //images:protokube.tar
+	bazel build ${BAZEL_CONFIG} --platforms=@io_bazel_rules_go//go/toolchain:linux_amd64 //protokube/cmd/protokube:image-bundle.tar
```

Comment:
This and the following hunk appear to be a separate concern, moving the location of the bazel build of protokube.

Reply:
I can move it in a separate PR if too much for this one. It is cleanup in the same department, removing everything from `/images`.

Reply:
A separate commit/explanation would have been good. I don't think it needs to be a separate PR.

Reply:
I can do that. Thanks :).

Reply:
Done.